.gitignore

```
@@ -7,9 +7,22 @@ package-lock.json
 # Testing
 /coverage
 
+# MAC
+.DS_Store
+
 # Production
 /build
 
+# Keys
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# MDB
+mdbreact-*
```

Please change to `mdb*`. Some MDB ressources have names like `mdb-sortable-*`
```suggestion
mdb*
```